src/app/_components/OrbitAmbassadorCard.tsx

```diff
@@ -0,0 +1,60 @@
+import clsx from 'clsx'
+
+import { BasicCard } from '@/components/BasicCard'
+import { StaticImage, type StaticImageProps } from '@/components/StaticImage'
+
+export type OrbitAmbassadorCardProps = {
+  index: number
+  image: StaticImageProps
+  name: string
+  testimonial: Array<string> | Array<JSX.Element>
+  location: string
+}
+
+import { buildImageSizeProp } from '@/utils/buildImageSizeProp'
+
+export function OrbitAmbassadorCard({
+  index,
+  image,
+  name,
+  testimonial,
+  location,
+}: OrbitAmbassadorCardProps) {
+  const isImagePositionOdd = index % 2 === 1
+
+  return (
+    <article
+      className={clsx(
+        'flex flex-col gap-4 lg:flex-row lg:gap-6',
```

Comment:
should be gap-6 (24px) according to designs

Reply:
I think this better groups the cards on smaller screens. I'll talk to Filipa. 🙏🏼